### ISSUE TITLE
Upgrade black dependency to 24.4.2?

### ISSUE THREAD

**Issue by vengroff** [inspera/blackbricks#60]
Would it be possible to upgrade the version of black to 24.4.2 and release a new version? That way it would work in environments that have moved up versions of black for other reasons.

I'd be happy to submit a PR for for this and also include the dependabot issues #56 and #57 just to get everything up to the latest. Let me know if you'd be interested in a PR like this and I'll submit one.

**Comment by bsamseth** [comment]
Thanks for offering to submit a PR. There's a few manual steps still to push out a new release, so I just did it now. I just needed a kick in the ass to actually do it. 👍🏻

**Comment by vengroff** [comment]
Thanks for doing this right away!

I actually jumped the gun a bit before you did and forked https://github.com/vengroff/blackbricks/tree/master and 
updated `black` plus some other dependencies. I ended up updating more things because I'm hoping to use this in
an environment that is currently quite up to date on most things.

This morning I pulled in your upstream changes. The fork is at https://github.com/vengroff/blackbricks/tree/master

I'm not sure what all your manual steps are, but I did run this against the test notebook in the repo and it worked. Let me know if this would be a worthwhile pull request.
